dbt/context/common.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, model, adapter, profile):
     def wrap_with_profile_and_model_name(self, fn):
         def wrapped(*args, **kwargs):
             args = (self.profile,) + args
-            kwargs['model_name'] = self.model.get('name')
+            kwargs['model_name'] = self.model.get('alias')
```

Comment:
I'm not sure we want to do this. The adapters use this `model_name` arg to name their database connection in the pool. I think this line might make it possible for two different models to grab the same connection, which would probably be undesirable.

Reply:
yeah i think this might be causing bugs on my branch...